CMakeLists.txt

```diff
@@ -1949,10 +1949,12 @@ elseif(APPLE)
             ${SECURITY_FRAMEWORK})
     endif()
 else()
-    # DH requires math (m) library
-    target_link_libraries(wolfssl
-        PUBLIC
-            m)
+    if(WOLFSSL_DH)
```

Comment:
Is there a way to also check for `!WOLFSSL_DH_CONST`?

Reply:
Added `WOLFSSL_DH_CONST` and logic matching configure.ac